### DISCUSSION TITLE
Image quality

### DISCUSSION THREAD

**Discussion by elminster13** [wictorwilen/ring-timelapse#11] [General]
Hi - thank you so much for sharing this project, I installed it on the weekend and it works perfectly. So helpful!!

I was wondering about the image quality, seems to be a bit low resolution; is there a way to change that?

**Comment by HZC-blip** [comment]
I am also having this issue. im going to attempt to set up a home assistant as that is also something the author was using in their blog post

**Comment by HZC-blip** [comment]
I just got done talking to ring and the snapshot images are small 640x360 so its ring itself and not something we can configure with a variable
